.github/workflows/release-wheels.yml

```diff
@@ -206,7 +225,8 @@ jobs:
         python -m pip uninstall -y pylibjpeg-libjpeg
         python -m pip uninstall -y pylibjpeg-openjpeg
         python -m pip install git+https://github.com/pydicom/pylibjpeg-data
-        python -m pip install -U --find-links dist/ pylibjpeg-libjpeg
+        ls -l dist
+        python -m pip install -U dist/*.whl
```

Comment:
I think this will not work as it will try to install all wheels, while with the old command it will look in that directory and find the matching one.

Reply:
I thought that it would be just 1 whl, but indeed we would have one per each platform all in there, uff.
I didn't like previous one since as we discovered, it is fine to find none and  just install from online while downloading that thing for matching version.  We do not want to temporarily boost version for testing here  since it might hide the issue later IMHO.  Ideally we should have got only that particular whl here but the day is short so I indeed will just kick this commit out.

Reply:
I think there is an option in pip to prevent it from looking online, I forgot what it's called (not at a computer right now).

Reply:
might be as generic as setup `http_proxy=http://127.0.0.1:9/ https_proxy=http://127.0.0.1:9/` env vars, but might be too brute force . may be @asmacdo or @jwodder know a more gentle way from top of their heads?

Reply:
The `--no-index` option to `pip install` will cause it to ignore the package index and only use `--find-links`.

Reply:
Exactly, that is what I meant, thanks.